dashboards-observability/public/plugin.ts

```diff
@@ -48,14 +48,16 @@ export class ObservabilityPlugin implements Plugin<ObservabilitySetup, Observabi
         const dslService = new DSLService(coreStart.http);
         const savedObjects = new SavedObjects(coreStart.http);
         const timestampUtils = new TimestampUtils(dslService);
+        const queryManagerInstance = new QueryManager();
```

Comment:
We don't have to call it explicitly xxxInstance, just rename it to queryManager or even simpler qm